packages/core/src/api.ts

```diff
@@ -18,13 +17,21 @@ function _getIngestEndpoint(dsn: DsnComponents): string {
 
 /** Returns a URL-encoded string with auth config suitable for a query string. */
 function _encodedAuth(dsn: DsnComponents, sdkInfo: SdkInfo | undefined): string {
-  return urlEncode({
```

Comment:
hmm wait did this get confused with another PR? This is the `validSeverityLevels` PR here, should this be in https://github.com/getsentry/sentry-javascript/pull/14406 instead? 😅

Reply:
yes 😶 I pushed all of these PRs within like 30s and messed everything up 😂